test/index.js

```diff
@@ -1908,6 +1908,39 @@ describe('expect()', () => {
                 Hoek.assert(exception.message === 'Expected \'a4x\' to match /\\w\\dy/', exception);
                 done();
             });
+
+            it('validates array assertion', (done) => {
+
+                const throws = function () {
+
+                    Code.expect(['zero', 'audience']).to.match(/roa/);
+                };
+
+                let exception = false;
+                try {
+                    Code.expect(throws).to.throw();
```

Comment:
This should just be `Code.expect(['zero', 'audience']).to.match(/roa/);`. Expecting it to throw makes the `try...catch` unnecessary.